### PR TITLE
Added Item Mods, removed some bad mods

### PR DESCRIPTION
**Pull request by R3P0FFXI**
Added multiple weapon and item mods, removed some bad mods

### CODE REVIEW

File: sql/item_mods.sql [hunk 1]
```diff
@@ -5954,9 +5958,9 @@ INSERT INTO `item_mods` VALUES (11925,26,9);
 INSERT INTO `item_mods` VALUES (11926,1,40);
 INSERT INTO `item_mods` VALUES (11926,10,25);
 INSERT INTO `item_mods` VALUES (11926,426,35);
-INSERT INTO `item_mods` VALUES (11927,1,40);
-INSERT INTO `item_mods` VALUES (11927,385,250);
-INSERT INTO `item_mods` VALUES (11927,831,-50);
+INSERT INTO `item_mods` VALUES (11927,1,40); -- def
+INSERT INTO `item_mods` VALUES (11927,385,250); -- shield bash
+INSERT INTO `item_mods` VALUES (11927,163,-50); -- Aegis (lv99 no afterglow) -50% Magic Damage Taken
```
Comment (R3P0FFXI) [inline sql/item_mods.sql]: @Hozu I added 831 back into Aegis but found it also assigned to MOD_Quick_Draw in status.lua

**Comment by R3P0FFXI** [comment]
@Hozu I added 831 back into Aegis but found it also assigned to MOD_Quick_Draw in status.lua

MOD_DMGMAGIC_II = 831 -- Magic Damage Taken II % (Aegis)
MOD_QUICK_DRAW_DMG_PERCENT = 831 -- Percentage increase to QD damage

Teo is currently moving QD to 580

File: sql/item_mods.sql [hunk 2]
```diff
@@ -22205,6 +22245,10 @@ INSERT INTO `item_mods` VALUES (18810,431,1);
 INSERT INTO `item_mods` VALUES (18811,370,1);
 INSERT INTO `item_mods` VALUES (18812,55,10);
 INSERT INTO `item_mods` VALUES (18820,288,3);
+INSERT INTO `item_mods` VALUES (18827,11,7); -- atk
```
Comment (takhlaq) [inline sql/item_mods.sql]: comment says attack mod says agi

**Comment by R3P0FFXI** [comment]
@takhlaq Mod was right comment was wrong

**Pull request merged**